GliaWidgets/SwiftUI/Components/SnackBarView/SnackBar.swift

```diff
@@ -0,0 +1,27 @@
+import Combine
+import SwiftUI
+import UIKit
+
+struct SnackBar {
+    func present(
+        text: String,
+        style: Theme.SnackBarStyle,
+        for presenter: UIViewController,
+        bottomOffset: CGFloat = 0,
+        timeProviding: FoundationBased.Timer.Providing = .live
```

Comment:
I would strongly suggest to avoid using `.live`  as default parameter. Instead I'd make required parameter. This will guarantee that some live side effect will not sneak in into unit test, causing flakiness.

Reply:
fixed - c635b8f06962f0641f9fcce4519b20d7c01e0dd5